rg_instructor_analytics/views/Suggestion.py

```diff
@@ -118,7 +118,7 @@ def generate_suggestion(self, course_key):
         def get_percent(total, put):
             return .0 if not (total and put) else float(put) / float(total)
 
-        subsections_percent = np.array([get_percent(u['student_count'], u['student_count_in']) for u in units])
+        subsections_percent = np.array([get_percent(u['student_count_in'], u['student_count']) for u in units])
```

Comment:
Why do we need numpy package here? I would use `unit` instead of `u` char. I missed that `conditions` is a function in the `filter_funnel`. Do we already have another `conditions` implementations?